tests/XRootD/http.sh

```diff
@@ -75,5 +75,66 @@ function test_http() {
 	       assert davix-rm "${HOST}/${TMPDIR}/${i}.ref"
 	done
 
+  # GET range-request
+	## Upload a file with a fixed content string
+	alphabetFile="alphabet.txt"
+	alphabetFilePath="${TMPDIR}/$alphabetFile"
+	echo -n "abcdefghijklmnopqrstuvw987" > $alphabetFilePath
+	assert curl -v -L -H 'Transfer-Encoding: chunked' "${HOST}/$alphabetFilePath" --upload-file $alphabetFilePath
+  outputFilePath=${TMPDIR}/output.txt
+  ## Download the file to a file and sanitize the output (remove '\r')
+  curl -v -L --silent -H 'range: bytes=0-3,24-26' "${HOST}/$alphabetFilePath" --output - | tr -d '\r' > $outputFilePath
+  ## Check the first content range header received
+  contentRange=`cat $outputFilePath | grep -i 'Content-range' | awk 'NR==1'`
```

Comment:
I suggest using `$( commands )` instead of back ticks, it's easier to read and allows nesting.